res/app/device-list/details/device-list-details-directive.js

```diff
@@ -277,6 +279,20 @@ module.exports = function DeviceListDetailsDirective(
         }
       })()
 
+      // check if new column needs a new scope or not
+      // and build accordingly
+      function buildColumn(col, device) {
```

Comment:
For performance reasons try to remove anything **angular**-related that goes to the device list.
If possible try to keep `angular-editable`, but it might be needed to write something like that from scratch and using only DOM if performance is not enough.